lld/ELF/Options.td

```diff
@@ -434,6 +434,9 @@ defm section_start: Eq<"section-start", "Set address of section">,
 
 def shared: F<"shared">, HelpText<"Build a shared object">;
 
+def randomize_section_padding: JJ<"randomize-section-padding=">,
+  HelpText<"Randomly insert padding between input sections using given seed">;
```

Comment:
While this can be simpler than ld.lld.1, I feel that segment padding should be mentioned.

Reply:
Done